citesphere/src/main/java/edu/asu/diging/citesphere/web/user/concepts/AddConceptController.java

```diff
@@ -49,12 +52,29 @@ public String show(Model model, Principal principal) {
 
     @RequestMapping(value = "/auth/concepts/add", method = RequestMethod.POST)
     public String post(@Validated @ModelAttribute("conceptForm") CitationConceptForm form, BindingResult result,
-            Model model, Principal principal) {
+            Model model, Principal principal, RedirectAttributes redirectAttributes) {
 
-        if (result.hasErrors()) {
+        if(result.hasErrors())
+        {
+            ListIterator<ObjectError> listIterator = result.getAllErrors().listIterator();
+            
+            while(listIterator.hasNext()) {
+                ObjectError error = listIterator.next();
+                
+                if(error.getDefaultMessage().equals("must be a valid URL")) {
```

Comment:
Other error messages would need to be displayed as well

Reply:
Makes sense, will make it generic